src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/ProcessRunner.cs

```diff
@@ -69,14 +69,8 @@ public async Task<bool> RunProcessAsync(ProcessInfo args, CancellationToken toke
 
             if (process.ExitCode != args.SuccessCode)
             {
-                if (args.Quiet)
-                {
-                    _logger.LogDebug("[{Tool}] Exited with non-success code: {ExitCode}", args.DisplayName, process.ExitCode);
-                }
-                else
-                {
-                    _logger.LogError("[{Tool}] Exited with non-success code: {ExitCode}", args.DisplayName, process.ExitCode);
-                }
+                var message = "[{Tool}] Error: Exited with non-success code: {ExitCode}";
+                _logger.Log(args.GetMessageLogLevel(message), message, args.DisplayName, process.ExitCode);
```

Comment:
nit: seems the reason *GetMessageLogLevel* exists is to parse the content from `e.Data`. Would prefer to use a LogError here since the *message* contains the string 'error'